core/validator.go

```diff
@@ -13,30 +13,29 @@ var (
 	ErrBlobQuorumSkip      = errors.New("blob skipped for a quorum before verification")
 )
 
-type ChunkValidator interface {
-	ValidateBatch([]*BlobMessage, *OperatorState, common.WorkerPool) error
-	ValidateBlob(*BlobMessage, *OperatorState) error
+type DataValidator interface {
```

Comment:
shall we create a name for data that's received by node (e.g. shard, BlobShard)? This naming may be more specific

Reply:
Are you suggesting changing `BlobMessage` into `BlobShard`?

Reply:
I'm suggesting "DataValidator" to something more specific such as "BlobShardValidator"

Reply:
DataMessageValidator?

Reply:
What about just `ShardValidator`?